pages/api/webhooks.js

```diff
@@ -85,7 +85,7 @@ const webhookHandler = async (req, res) => {
         }
       } catch (error) {
         console.log(error);
-        return res.json({ error: 'Webhook handler failed. View logs.' });
+        return res.status(400).send('Webhook error: "Webhook handler failed. View logs."');
```

Comment:
Any reason for text response instead of JSON?

Reply:
Only because that's what is being returned in the other error handler (line 49).

Reply:
Sorry, but for webhooks we don't want to send a 400 status because that causes Stripe to endlessly resend the webhook which we don't want.

Reply:
Sorry, but resending the response is exactly what is needed. More details below…